flink-table/flink-table-planner/pom.xml

```diff
@@ -301,6 +301,13 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>com.jayway.jsonpath</groupId>
+			<artifactId>json-path</artifactId>
+			<version>${jsonpath.version}</version>
+			<scope>test</scope>
```

Comment:
This seems incorrect. If it has a test scope then the loader won't bundle it, nor will it ever make it's way to the kafka tests (since test dependencies aren't transitive).

Reply:
Okay gotcha, i've removed the test scope. However, i am no longer able to get the `KafkaChangelogTableITCase` test running in the IDE anymore, even after rebuilding the modules on the command line. I have tried adding it individually and in various combinations in `flink-table-planner-loader-bundle, flink-table-planner-loader, flink-table-planner_${scala.binary.version}`. Did you have to do anything else to get it to work?

Reply:
actually wait a second, i was only rebuilding the flink-table-planner module, i will try also rebuilding the other modules

Reply:
Nvm that wasn't it, i had to add `<include>com.jayway.jsonpath:json-path</include>` to get it to work

Reply:
CI failed with 
```
Dependency convergence error for com.jayway.jsonpath:json-path:jar:2.4.0:runtime paths to dependency are:
+-org.apache.flink:flink-table-planner_2.12:jar:1.17-SNAPSHOT
  +-org.apache.calcite:calcite-core:jar:1.29.0:compile
    +-com.jayway.jsonpath:json-path:jar:2.4.0:runtime
and
+-org.apache.flink:flink-table-planner_2.12:jar:1.17-SNAPSHOT
  +-com.jayway.jsonpath:json-path:jar:2.6.0:compile
```
maybe i need to try adding the dependency in the `flink-table-planner-loader` pom instead?

Reply:
Add an entry to the dependencyManagement of the flink-table module for json-path.

Reply:
_or_ use json-path 2.4.0 (because that's what calcite expects anyway). In fact, we don't need the additional dependency in the pom; the include alone would solve this.

We should use a different relocation pattern though to ensure this doesn't clash with the json-path dependency of the table runtime.